Twoliter.toml

```diff
@@ -1,5 +1,5 @@
 schema-version = 1
-release-version = "4.1.0"
+release-version = "5.0.0"
```

Comment:
nit: Version bumps are typically (though not always!) done in a separate commit.

This otherwise LGTM!